baselines/diabetic_retinopathy_detection/README.md

```diff
@@ -27,91 +27,61 @@ Bayesian deep learning seeks to equip deep neural networks with the ability to p
 Set up and activate the Python environment by executing
 
 ```
-conda create -n ub python=3.8
+conda create -n ub python=3.10
 conda activate ub
 python3 -m pip install -e .[models,jax,tensorflow,torch,retinopathy]  # In uncertainty-baselines root directory
-pip install "git+https://github.com/google-research/robustness_metrics.git#egg=robustness_metrics"
-pip install 'git+https://github.com/google/edward2.git'
 ```
 
 ## Data Installation
 
-Because the data is distributed through Kaggle, we need to take a manual route to downloading.
+The below instructions will install and preprocess the data needed to train and evaluate models on the Country and Severity Shifts.
 
-1. Download from Kaggle: https://www.kaggle.com/c/diabetic-retinopathy-detection
+The EyePACS dataset (used in Country and Severity Shift) and the APTOS 2019 dataset (used in Country Shift) are distributed through Kaggle, which requires us to manually download the data and place it in the correct directory.
 
-2. Extract everything to ``$DATA_DIR/downloads/manual``; your directory should look like
+1. Download the raw datasets from Kaggle:
+   * [EyePACS](https://www.kaggle.com/c/diabetic-retinopathy-detection)
+   * [APTOS 2019](https://www.kaggle.com/c/aptos2019-blindness-detection)
 
-``sample/  sampleSubmission.csv  test/  train/  trainLabels.csv``
+2. Extract the EyePACS dataset to ``$DATA_DIR/ub_diabetic_retinopathy_detection/manual``. The directory structure should look like this:
 
-3. Confirm successful download of files
+    ``sample/  sampleSubmission.csv  test/  train/  trainLabels.csv``
 
-You should have 35,126 training images and 53,576 test images, which should be located in manual/train and manual/test.
+3. Extract the APTOS dataset to ``$DATA_DIR/aptos/manual``. The directory structure should look like this:
 
-You may check this with the command 
+    ``sample_submission.csv  test.csv  test_images/  train.csv  train_images/``
 
-`ls -1 | wc -l`
+4. Confirm successful download of files. The following commands should print out the number of files in the directories:
+    ```
+    $ ls -1 $DATA_DIR/ub_diabetic_retinopathy_detection/manual/train | wc -l
+    35126
+    $ ls -1 $DATA_DIR/ub_diabetic_retinopathy_detection/manual/test | wc -l
+    53576
+    $ ls -1 $DATA_DIR/aptos/manual/train_images | wc -l
+    3662
+    $ ls -1 $DATA_DIR/aptos/manual/test_images | wc -l
+    1928
+    ```
 
-4. Manual loading -- this is not contained in standard execution of diabetic-retinopathy model execution (yet)
+5. Manual shuffling and packaging of TF dataset objects. Consider running the following commands in a `tmux` or `screen` session, in case of a network failure. They take a while. 
+    ```
+    conda activate ub
+    
+    # Standard EyePACS dataset
+    python baselines/diabetic_retinopathy_detection/data_load_scripts/load_ub_diabetic_retinopathy_detection.py --data_dir=$DATA_DIR
```

Comment:
I think this section is stale now. Please update using the new data preparation script.